Detectors/MUON/MCH/Calibration/src/pedestals-decoding-workflow.cxx

```diff
@@ -145,7 +146,11 @@ class PedestalsTask
     auto mapFECfile = ic.options().get<std::string>("fec-map");
     initFee2SolarMapper(mMapCRUfile);
     initElec2DetMapper(mMapFECfile);
-
+    auto stop = [this]() {
+      LOG(INFO) << "time spent for decoding (ms): min=" << mTimeDecoderMin->count() << ", max="
+          << mTimeDecoderMax->count() << ", mean=" << mTimeDecoder.count() / mTFcount;
```

Comment:
do I read correctly that this will break when running with mDebug=false as mTFcount will be zero then ?

Reply:
Good catch! I will put the mTFcout increment outside of the mDebug block.